generate_workspace/src/main/java/com/google/devtools/build/workspace/output/BzlWriter.java

```diff
@@ -32,27 +35,33 @@
       MethodHandles.lookup().lookupClass().getName());
 
   private final String[] argv;
-  private final Path generatedFile;
+  private final File generatedFile;
```

Comment:
It was unable to create the file at `root/child/directory/generate_workspace.bzl`.
It would attempt to create the file at 
```
bazel-out/local-fastbuild/bin/.../BazelWriterTest.runfiles/__main__/
```
Then it would find that `root/child/directory/generate_workspace.bzl` was not created.

The IOException occurred when trying to create the PrintStream.